src/interfaces/IChainlinkOracle.sol

```diff
@@ -5,6 +5,10 @@ import {IERC4626} from "./IERC4626.sol";
 import {AggregatorV3Interface} from "./AggregatorV3Interface.sol";
 import {IOracle} from "../../lib/morpho-blue/src/interfaces/IOracle.sol";
 
+/// @title IChainlinkOracle
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Interface exposed by the ChainlinkOracle.
```

Comment:
```suggestion
/// @notice Interface of ChainlinkOracle.
```
[optional] to be consistent with the phrasing of the bundler (also technically ChainlinkOracle could expose different interfaces)